### ISSUE TITLE
Additional formats

### ISSUE THREAD

**Issue by galad87** [SublerApp/Subler#5]
Many codes have official specifications for the ISO File Format, but aren't supported yet in Subler

- [ ] Flac
- [ ] Opus
- [ ] TrueHD
- [ ] DTS

**Comment by Mock1** [comment]
@galad87 This is meant for support for Passthru, right? We can convert these down, but we can't pass them through.
This will require changes to MP4v2, right? I tried to get a TrueHD passthru to work with Subler, but the output file was garbage.
It should also be possible to use FFMPEG to mux those tracks back into the output of MP4v2's muxing. But the built out FFMPEG that is in contrib doesn't include any of the encoding pieces.

I *really* want to add passthru support for TrueHD, but I don't understand MP4v2 enough to contribute to it.

**Comment by galad87** [comment]
Right. mp4v2 needs some addition to write the extradata, as defined in https://developer.dolby.com/globalassets/technology/dolby-truehd/dolby-truehd-mlp-bitstreams-within-the-iso-base-media-file-format.pdf I suppose.

**Comment by galad87** [comment]
Yes, I compile it from https://github.com/SublerApp/mp4v2

**Comment by Salem874** [comment]
Passthrough does work when opening an MP4 file in Subler, but not when opening an MKV in Subler.

Opening an MKV defaults converting to AAC, and the Passthrough option is disabled.

Passing/muxing a TrueHD into MP4 (as evidenced when opening an MP4 with TrueHD) results in a usable MP4, EXCEPT in Quicktime or anything Apple-related, as sthey for some reason dont like MP4 with TrueHD. All other players (VLC, MPC-HC etc.) that i've tried are able to play the file.
